packages/braid-design-system/src/lib/components/Toggle/Toggle.tsx

```diff
@@ -56,25 +57,21 @@ export const Toggle = forwardRef<HTMLInputElement, ToggleProps>(
     const lightness = useBackgroundLightness();
 
     if (process.env.NODE_ENV !== 'production') {
-      if (bleedY === undefined) {
+      if (_bleedY !== undefined) {
```

Comment:
I typically favour using `typeof` checks over identity checks:

```suggestion
      if (typeof _bleedY !== 'undefined') {
```

[MDN says]:
> For checking potentially non-existent variables that would otherwise throw a [ReferenceError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError), use typeof nonExistentVar === "undefined" because this behavior cannot be mimicked with custom code.

[MDN says]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#:~:text=For%20checking%20potentially%20non%2Dexistent%20variables%20that%20would%20otherwise%20throw%20a%20ReferenceError%2C%20use%20typeof%20nonExistentVar%20%3D%3D%3D%20%22undefined%22%20because%20this%20behavior%20cannot%20be%20mimicked%20with%20custom%20code.

Reply:
ahh clever!